lib/membrane_opus.ex

```diff
@@ -20,12 +20,9 @@ defmodule Membrane.Opus do
 
   @type t :: %__MODULE__{
           bitrate: bitrate_t,
-          channels: channels_t
+          channels: channels_t,
+          self_delimiting?: boolean
         }
 
-  @enforce_keys [
-    :bitrate,
-    :channels
-  ]
-  defstruct @enforce_keys
+  defstruct [:bitrate, :channels, self_delimiting?: false]
```

Comment:
Shouldn't there be any defaults?

Reply:
The defaults are `nil`, meaning they are unknown